src/main/java/Duke.java

```diff
@@ -1,10 +1,105 @@
+    private static int taskCount;
+    private final static int MAX_TASK = 100;
+    private static Task[] taskList = new Task[MAX_TASK];
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        System.out.println("\t____________________________________________________________");
+        System.out.println("\tHello! I'm Duke");
+        System.out.println("\tWhat can I do for you?");
+        System.out.println("\t____________________________________________________________");
+        Duke.executeCommand();
+    }
+
+    // Recognise command passed to Duke
+    public static void executeCommand() {
+        // Execute different response for different types of command
+        String line;
+        Scanner in = new Scanner(System.in);
+        int taskNumber;
+        while (true) {
+            line = in.nextLine();
+            if (line.equalsIgnoreCase("bye")) {
+                break;
+            }
+            if (line.equalsIgnoreCase("list")) {
+                displayList();
+                continue;
+            }
+            if (line.contains("done")) {
+                taskNumber = Integer.parseInt(line.replaceAll("[^\\d]", " ").trim());
+                taskList[taskNumber - 1].setDone();
+                continue;
+            }
+            storeText(line);
+        }
+
+        System.out.println("\t____________________________________________________________");
+        System.out.println("\t Bye. Hope to see you again soon!");
+        System.out.println("\t____________________________________________________________");
+    }
+
+    public static void storeText(String text) {
+        String task;
+        String at;
+        String by;
+
+        //Assign deadline task
+        if(text.startsWith("deadline")){
+            task = text.substring(9,text.indexOf(" \\by"));
+            by = text.substring(text.indexOf("\\by") + 4, text.length());
+            taskList[taskCount] = new Deadline(task, by);
+            printTaskAssignment(taskCount);
+            taskCount++;
+
+        }   else if(text.startsWith("event")){
+            //Assign event task
+            task = text.substring(6,text.indexOf(" \\at"));
+            at = text.substring(text.indexOf("\\at") + 4, text.length());
+            taskList[taskCount] = new Event(task, at);
+            printTaskAssignment(taskCount);
+            taskCount++;
+
+        }   else if (text.startsWith("todo")){
+            //Assign todos task
+            task = text.replace("todo ", "");
+            taskList[taskCount] = new Todo(task);
+            printTaskAssignment(taskCount);
+            taskCount++;
+        }
+    }
+
+    public static void displayList() {
```

Comment:
The name for this method could be renamed to printList to help ensure consistency with printTaskAssignment